src/Payloads/MarkdownPayload.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace LaraDumps\LaraDumps\Payloads;
+
+use Illuminate\Mail\Markdown;
+
+class MarkdownPayload extends Payload
+{
+    public function __construct(
+        public string $dump
+    ) {
+    }
+
+    public function type(): string
+    {
+        return 'dump';
```

Comment:
Shouldn't this be 'html' here? It might be broken to send straight to 'dump'

Reply:
I tested it & working fine (test case included). 

If we're using `html` (or something) than an additional payload must be added.